xtask/src/transfer_workload.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{
+    backend::{Action, DB},
+    timer::Timer,
+};
+
+pub fn transfer_workload(
+    mut backend: Box<dyn DB>,
+    n_accounts: u64,
+    additional_initial_capacity: u64,
+    timer: &mut Timer,
+) {
+    let accounts = (0..n_accounts * 2)
```

Comment:
This creates twice as many accounts as the parameter. The caller might be confused as a result.

Reply:
Oh, I misnamed the argument of the function. It should not be `n_accounts` but rather the `size` of the workload. Therefore, considering that a transfer workload involves 2 reads and 2 writes between two accounts (based on what can be done in the database, only perform a vector of actions), I believed we could achieve a transfer workload by transferring balance between `size_transfer_workload * 2` accounts